core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/script/dto/ScriptDtoRepresentationModel.java

```diff
@@ -0,0 +1,36 @@
+package io.metadew.iesi.server.rest.script.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.hateoas.Links;
+import org.springframework.hateoas.RepresentationModel;
+import org.springframework.lang.NonNull;
+
+public class ScriptDtoRepresentationModel<T extends ScriptDtoRepresentationModel <? extends T>> extends RepresentationModel<T> {
```

Comment:
Maybe rename this to 'NoEmptyLinksRepresentationModel'. This way we can reuse it for other models as well